### ISSUE TITLE
different color for building=construction

### ISSUE THREAD

**Issue by maro-21** [gravitystorm/openstreetmap-carto#4164]
How about rendering `building=construction` in a different color? For example a lighter shade of gray, or with an edge in a different color.

**Comment by HolgerJeromin** [comment]
ref #2532

**Comment by jeisenbe** [comment]
I've changed the title to "Distinctive rendering for building=construction" to more clearly state the requested new feature. 

Currently buildings under construction are rendered the same as all other buildings if they are tagged as `building=construction`. Using a lifecycle prefix like `construction:building=*` leds to no rendering of the feature.

We could consider adding a distinctive rendering for the established tag `building=construction`, if anyone has a good idea about how it could look.

**Comment by polarbearing** [comment]
We have a very distinctive rendering for ```landuse=construction```, which usually surrounds the constructed building. Do we need a distinction for the building itself then?

**Comment by maro-21** [comment]
> We have a very distinctive rendering for landuse=construction, which usually surrounds the constructed building. Do we need a distinction for the building itself then?

There may be situations when there is no construction site around and the building is not finished, sometimes for many years.

**Comment by Adamant36** [comment]
I don't think it would be good to render building=construction in a unique way. Even if it has the usage numbers. A house is still a house even if it's not completely done being built yet and "construction" isn't really a building type. Is it more important that map consumers know a plot of land has a house on it? or that there is a general, undefined, type of undefined building that renders a certain way because the carpeting isn't done being stapled? 

Also, it's not really clear when construction is over and a different building tag should be used instead. Many hotels are in an almost constant state re-modeling/construction/renovation. So should the whole place be rendered or tagged differently because of it while other parts of the hotel are still being used as a hotel? Again, does it serve map consumers more to know that the place is a hotel or some "under construction" building? What's the threshold before it isn't under construction anymore? 

Those issues aren't really present with landuse=construction IMO. It's pretty clear when a construction zone is gone and it is more of an actual type of landuse then "construction" is a type of building.

**Comment by jeisenbe** [comment]
This is a well-established tag, used over 0.5 million times. Generally it is used for buildings that are not yet finished and cannot be used, because they lack complete walls or roofs, so are clearly construction sites. This makes it a verifiable tag. While this state is usually temporary, construction can take years.

It is worth considering rendering these, because a building under construction might not qualify as a `building=` feature normally, due to incomplete walls and roof.

(Similarly, building=roof would be the other tag that would most benefit from a distinctive rendering, since walls are lacking).

**Comment by IloveRumania** [comment]
I think they should have a dashed outline.
Looking at a construction area with a mix of completed buildings and under-construction buildings in Carto, it's impossible to tell which ones are fully built versus which ones are still being built.
